markdown/api-docs/cart-and-checkout/embedded-checkout.md

```diff
@@ -0,0 +1,335 @@
+<h1>Embedded Checkout</h1>
+<div class="otp" id="no-index">
+	<h3> On This Page </h3>
+	<ul>
+    <li><a href="#embedded-checkout_prerequisites">Step 1: Prerequisites</a></li>
+    <li><a href="#embedded-checkout_create-channel">Step 2: Create a Channel</a></li>
+    <li><a href="#embedded-checkout_create-site">Step 3: Create a Site</a></li>
+    <li><a href="#embedded-checkout_create-cart">Step 4: Create a Cart</a></li>
+    <li><a href="#embedded-checkout_create-routes">Step 5: Create Routes</a></li>
+    <li><a href="#embedded-checkout_embed-checkout">Step 6: Embed Checkout</a></li>
+	</ul>
+</div>
+
+Embedded Checkout lets you place BigCommerce’s checkout onto any website. Customers can check out on an external storefront while their order information syncs simultaneously to the BigCommerce Control Panel. You can see this in action within the BigCommerce for WordPress plugin, which uses the same process described here as a checkout option for merchants. View more information about the plugin in the article [BigCommerce for Wordpress](https://developer.bigcommerce.com/bigcommerce-for-wordpress/getting-started/introduction).
+
+This tutorial will walk you through creating a working BigCommerce embedded checkout example deployed to [Heroku](https://heroku.com). Embedded checkout uses an iFrame to insert the Optimized One-Page Checkout from your BigCommerce store into another site. This is a good solution if you want to sell products from your BigCommerce store on another site without worrying about PCI compliance.
+
+You will need to use the [Channels, Sites and Routes](#) APIs to complete this tutorial. The Channels API allows you to create and manage sales channel listings across a merchant's channels. The Sites and Routes APIs let you set an external storefront domain and destination routes that are not BigCommerce URLs.
```

Comment:
Suggest changing:

> The Channels API allows you to create and manage sales channel listings across a merchant's channels

To this:
The Channels API allows you to create and manage sales channel listings across a merchant's product catalog. A channel can be a marketplace, like Amazon, or an external storefront, like a WordPress site.

.